paper-ieee-trans.tex

```diff
@@ -57,19 +55,18 @@ \section{Introduction}
 supporting unified access to large--scale computing and intense data analysis
 applications, has been driven by the needs of scientific communities as part of their
 pan--European research collaborations. One such example is the Worldwide
-Large Hadron Collider (LHC) Computing Grid infrastructure, driven
-by the particle physics scientific community, built to satisfy the computing needs
+Large Hadron Collider (LHC) Computing Grid infrastructure: driven
+by the particle physics scientific community; built to satisfy the computing needs
```

Comment:
-1

Reply:
+1 It is a way to read the sentence clearer than now.

Reply:
sorry no
this was reviewed